cinema/models.py

```diff
@@ -134,9 +137,8 @@
         )
 
     def __str__(self):
-        return (
-            f"{str(self.movie_session)} (row: {self.row}, seat: {self.seat})"
-        )
+        return (f"{str(self.movie_session)} "
+                f"(row: {self.row}, seat: {self.seat})")
 
     class Meta:
         unique_together = ("movie_session", "row", "seat")
```

Comment:
The use of `unique_together` is a great way to ensure that a ticket can't be created with the same `movie_session`, `row`, and `seat`. However, note that Django has started favoring the use of `UniqueConstraint` which allows for more flexibility. Consider switching to `UniqueConstraint` in the future.